hazelcast/src/test/java/com/hazelcast/journal/AbstractEventJournalBasicTest.java

```diff
@@ -71,9 +72,9 @@
     private int partitionId;
     private TruePredicate<EJ_TYPE> TRUE_PREDICATE = new TruePredicate<EJ_TYPE>();
     private Projection<EJ_TYPE, EJ_TYPE> IDENTITY_PROJECTION = Projections.identity();
+    private boolean loadAllPublishesAdded = false;
 
-    @Before
-    public void init() {
+    private void init() {
```

Comment:
Removed `@Before` from `init() so that the test methods can adjust the instance config.

Reply:
Looks messy, probably it worth to factor the two dependent methods into a separate test case and make it `Parameterized` since the code is almost identical.

Reply:
Yes, it's messy. Since the config needs to be adjusted from inside the test cases, either this hack, or we need to copy the (map) test class hierarchy only for one of the three `LOADED` cases and make it `Parameterized`. I was thinking in the way that this hack is acceptable in favor of keeping the journal tests together. No strong preference though :)

The common code of the two `loadAll` cases can be extracted even inside this test class. It is worth doing it even if we accept the commented hack.

Reply:
Ah, I see, there is a lot of "private" code to extract, let's keep it as it is then.